.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,55 @@
+name: Bug Report
+description: If something isn't working the way you expect it to.
+labels:
+  - needs triage
+body:
+  - type: markdown
+    attributes:
+      value: Thanks for taking the time to fill out this bug report!
+
+  - type: dropdown
+    id: sdk
+    attributes:
+      label: SDK
+      description: What SDK are you running?
+      options:
+        - Node
+        - PHP
+        - Python
+        - Ruby
+        - .NET
+      default: 0
+    validations:
+      required: true
+
+  - type: input
+    id: version
+    attributes:
+      label: Which version?
+      description: Which version of that SDK are you using?
+    validations:
+      required: true
+
+  - type: textarea
+    id: use-case
+    attributes:
+      label: What's your use case?
+      description: Tell us how you use ReadMe Metrics!
```

Comment:
a million dollar question for maybe @gratcliff — is this what we should be referring to the product as?

Reply:
😅 sounds like you're already on top of it! @kanadgupta agreed that this should be changed to "Developer Dashboard", but we'll need to update that copy throughout this repo. I can assign this to myself in the short term!

Reply:
In the meantime, this makes sense to me!